.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -235,7 +229,13 @@ jobs:
         curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin
 
         trivy image --ignore-unfixed --no-progress --severity HIGH,CRITICAL,MEDIUM --exit-code 1 ${{ variables.repoImageName }}:$(linuxImagetag)
-
+  
+  - bash: |
```

Comment:
FYI, Golang unit tests are failing which needs to be fixed.

Reply:
yes